lib/parser.js

```diff
@@ -1,8 +1,8 @@
 'use strict'
 
 var ReplyError = require('./replyError')
-// TODO: Consider shrinking the bufferPool if it's not used a lot by using interval check
 var bufferPool = new Buffer(64 * 1024)
+var interval = null
```

Comment:
Wouldn't this and `bufferPool` cause inconsistencies if running more than one parser as they are module scoped. Perhaps move into constructor?